cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -85,11 +85,19 @@ def internal_notes(self, key, value):
     return {"value": clean_val("a", value, str, req=True)}
 
 
-@model.over("note", "^935__")
+@model.over("note", "(^935__)|(^866__)")
 @out_strip
 def note(self, key, value):
     """Translates note field."""
-    return clean_val("a", value, str, req=True)
+    notes_list = [self.get("note", "")]
+    if key == "935__":
```

Comment:
you could decrease the amount of code by removing this if statement (and the one below for checking if it is 866), since you are reading `a` subfield in each case anyway